common/authenticator-requests/src/v4/conversion.rs

```diff
@@ -137,25 +140,51 @@ impl TryFrom<v4::response::AuthenticatorResponse> for v3::response::Authenticato
         Ok(Self {
             data: authenticator_response.data.try_into()?,
             reply_to: authenticator_response.reply_to,
-            protocol: authenticator_response.protocol,
+            protocol: Protocol {
+                version: 3,
+                service_provider_type: authenticator_response.protocol.service_provider_type,
+            },
         })
     }
 }
 
+impl TryFrom<v3::response::AuthenticatorResponseData> for v4::response::AuthenticatorResponseData {
+    type Error = crate::Error;
+    fn try_from(
+        authenticator_response_data: v3::response::AuthenticatorResponseData,
+    ) -> Result<Self, Self::Error> {
+        match authenticator_response_data {
+            v3::response::AuthenticatorResponseData::PendingRegistration(_) => Err(
+                Self::Error::Conversion("mac hash breaking change".to_string()),
```

Comment:
what has actually changed in the mac? is there no way to backport the changes? I wonder if this might cause client breakage

Reply:
The ipv6 was added to the payload. It's not possible to convert those messages without the private key. It's better to actually remove the conversion, because it forces the re-creation of the messages via the `new` function, which requires the private key and re-does the mac.